### PR TITLE
Update projects.json

### PR DESCRIPTION
**Pull request by austriker27**
adding two websites made with gridsome to this list!

### CODE REVIEW

File: projects.json [hunk 1]
```diff
@@ -157,5 +157,17 @@
...
+  },
+  {
+  "name": "The Lockdown",
+  "description": "A Fan Written Seinfeld script during COVID-19",
+  "url": "https://thelockdown.netlify.app",
+  "tags": ["script", "tailwindcss", "seinfeld", "tv show"]
+},
+  {
+  "name": "lindahl studios",
+  "description": "a small two man/brother development shop",
```
Comment (HZooly) [inline projects.json:169]: Need capitalized description

**Comment by HZooly** [comment]
Hi @austriker27, thanks for these two sites. Can you please check the review? Just two little things missing

**Comment by austriker27** [comment]
thanks @HZooly I made the formatting updates. (Sorry the lindahl studios llc is technically lower case so thats why I typed it that way)

**Comment by HZooly** [comment]
@austriker27 thanks for the fixes 😃

**Pull request merged**